analytics.go

```diff
@@ -6,7 +6,7 @@ import (
 	"regexp"
 	"time"
 
-	"github.com/jeffail/tunny"
+	"github.com/Jeffail/tunny"
```

Comment:
yeah, build is failing because of that change. I need to investigate why it became capitalized